stub-runner/stub-runner-spring/src/main/groovy/com/ofg/stub/config/StubRunnerConfiguration.groovy

```diff
@@ -54,7 +53,7 @@ import org.springframework.context.annotation.Import
  * @see ServiceConfigurationResolver
  */
 @Configuration
-@Import([CustomZookeeperDiscoveryClientConfiguration, ZookeeperDiscoveryAutoConfiguration, ServiceDiscoveryTestingServerConfiguration])
```

Comment:
`CustomZookeeperDiscoveryClientConfiguration` removed (created because of update Camden.SR7 to Dalstor.SR4), `ZookeeperDiscoveryAutoConfiguration` provided by `ZookeeperServiceResolverConfiguration`.